ui/pages/bridge/awaiting-signatures/bridge-step-icon.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function BridgeStepIcon({ stepNumber = 1 }) {
```

Comment:
Minor: [AvatarBase](https://metamask.github.io/metamask-storybook/?path=/docs/components-componentlibrary-avatarbase--docs) can be used to show the numbers as well

Reply:
Thanks for the tip, done!